packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -177,3 +189,131 @@ test.describe('severity-1 #smoke', () => {
     });
   });
 });
+
+test.describe('severity-2 #smoke', () => {
+  test.describe('signup react', () => {
+    test('signup invalid email', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      email = 'invalid';
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await expect(
+        page.getByText('Valid email required', { exact: true })
+      ).toBeVisible();
+    });
+
+    test('empty email', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst('');
+      await expect(
+        page.getByText('Valid email required', { exact: true })
+      ).toBeVisible();
+    });
+
+    test('coppa is too young', async ({ page, pages: { signupReact } }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await signupReact.fillOutSignupForm(PASSWORD, '12');
+      await page.waitForURL(/cannot_create_account/);
+    });
+
+    test('Visits the privacy policy links save information upon return', async ({
+      page,
+      pages: { signupReact },
+    }) => {
+      skipCleanup = true;
+      await signupReact.goto();
+      await signupReact.fillOutEmailFirst(email);
+      await signupReact.fillOutSignupForm(PASSWORD, '21', false);
+      await signupReact.visitPrivacyPolicyLink();
+      await page.waitForURL(/legal\/privacy/);
+      await page.goBack();
+
+      // TBD: Currently not supported. Do we need to support this?
```

Comment:
I would run this one by Vesta.

Reply:
https://mozilla-hub.atlassian.net/browse/FXA-8797